src/Neo/IO/Caching/ReflectionCache.cs

```diff
@@ -32,11 +34,13 @@ static ReflectionCache()
                 if (attribute == null) continue;
 
                 // Append to cache
-                s_dictionary.Add((T)field.GetValue(null), attribute.Type);
+                var key = field.GetValue(null) as T;
```

Comment:
```suggestion
                var key = (T?)field.GetValue(null);
```
Shouldn't  use `?`. Because, the value will be the underlying type of th `enum`. Enums are not types or classes. There like a structure. 

Sorry didnt know `T` as only a `enum`.